content/courses/intro-to-ada/chapters/appendices.rst

```diff
@@ -155,52 +155,52 @@ Indefinite version
-| :ada:`type T (<>) is abstract tagged limited private;`       | abstract or concrete    |
-+--------------------------------------------------------------+-------------------------+
-| Indefinite derived tagged type                               | Any tagged type derived |
-|                                                              | from base type B,       |
-| **Format**: :ada:`type T (<>) is new B with private;`        | indefinite or definite  |
-+--------------------------------------------------------------+-------------------------+
-| Indefinite abstract derived tagged type                      | Any tagged type derived |
-|                                                              | from base type B,       |
-| **Format**:                                                  | indefinite or definite  |
-| :ada:`type T (<>) is abstract new B with private;`           | abstract or concrete    |
-+--------------------------------------------------------------+-------------------------+
++--------------------------------------------------------------+--------------------------+
+| Formal type                                                  | Actual type              |
++==============================================================+==========================+
+| Indefinite incomplete type                                   | Any type                 |
+|                                                              |                          |
+| **Format**: :ada:`type T (<>);`                              |                          |
++--------------------------------------------------------------+--------------------------+
+| Indefinite nonlimited private type                           | Any nonlimited type      |
+|                                                              | indefinite or definite   |
+| **Format**: :ada:`type T (<>) is private;`                   |                          |
++--------------------------------------------------------------+--------------------------+
+| Indefinite limited private type                              | Any type, limited or     |
+|                                                              | not, indefinite or       |
+| **Format**: :ada:`type T (<>) is limited private;`           | definite                 |
++--------------------------------------------------------------+--------------------------+
+| Incomplete indefinite tagged private type                    | Any concrete tagged      |
+|                                                              | type,                    |
+| **Format**: :ada:`type T (<>) is tagged;`                    | indefinite or definite   |
++--------------------------------------------------------------+--------------------------+
+| Indefinite tagged private type                               | Any concrete, nonlimited |
+|                                                              | tagged type,             |
```

Comment:
Editorial remark: instead of changing the whole table just because of a single change, I'd really prefer you just add a new line (after "Any concrete" and before "tagged type") to write "nonlimited".

Reply:
:+1:   it didn't cross my mind